README.md

```diff
@@ -2,27 +2,28 @@
-| [github__weekly_metrics](https://dbt-github.netlify.app/#!/model/model.github.github__weekly_metrics)    | Each record represents a single week, enriched with metrics about PRs and issues that were created and closed during that period.                             |
-| [github__monthly_metrics](https://dbt-github.netlify.app/#!/model/model.github.github__monthly_metrics)   | Each record represents a single month, enriched with metrics about PRs and issues that were created and closed during that period.                            |
-| [github__quarterly_metrics](https://dbt-github.netlify.app/#!/model/model.github.github__quarterly_metrics) | Each record represents a single quarter, enriched with metrics about PRs and issues that were created and closed during that period.                          |
+| [github__pull_requests](https://dbt-github.netlify.app/#!/model/model.github.github__pull_requests)     | Each record represents a GitHub pull request, enriched with data about its repository and reviewers, as well as the durations between review requests, merges, and reviews. |
+| [github__daily_metrics](https://dbt-github.netlify.app/#!/model/model.github.github__daily_metrics)     | Each record represents a single day, enriched with metrics about PRs and issues that were created and closed that day.                              |
+| [github__weekly_metrics](https://dbt-github.netlify.app/#!/model/model.github.github__weekly_metrics)    | Each record represents a single week, enriched with metrics about the PRs and issues that were created and closed that week.                             |
+| [github__monthly_metrics](https://dbt-github.netlify.app/#!/model/model.github.github__monthly_metrics)   | Each record represents a single month, enriched with metrics about PRs and issues that were created and closed that month.                            |
+| [github__quarterly_metrics](https://dbt-github.netlify.app/#!/model/model.github.github__quarterly_metrics) | Each record represents a single quarter, enriched with metrics about PRs and issues that were created and closed that quarter.                          |
 
-# 🤔 Who is the target user of this dbt package?
-This package is intended to be leveraged by individuals who want to take advantage of analysis ready models to fuel their reporting needs. The package outputs reporting tables that may be easily linked directly to your BI tool of choice for quick time to value with your GitHub data.
+# 🤔 Why should I use this dbt package?
+This package is designed for users who want to use pre-built, analysis-ready models to fuel their reporting needs. The package creates reporting tables that you link directly to your BI tool for quick time to value with your GitHub data.
```

Comment:
Is there another way to say “time to value”? It’s an expression that originated in the North American tech space, and I’m not sure how global its usage is.

Reply:
We could always just stick with "quick start". Thoughts?

Reply:
@fivetran-joemarkiewicz I worry about using "quickstart" - we plan to use that word to refer to the Quickstart Data Models feature in the dashboard. What about "...directly to your BI tool so you can quickly derive value from your GitHub data"? Is that too wordy?